app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/common/AlertDialogs.kt

```diff
@@ -39,6 +39,38 @@ fun ErrorAlertDialog(
     )
 }
 
+@Composable
+fun ConfirmAlertDialog(
+    title: String,
+    message: String,
+    positiveBtnText: String,
+    negativeBtnText: String,
```

Comment:
Missed two 😄 
```suggestion
    positiveButtonText: String,
    negativeButtonText: String,
```

Reply:
@AVI5HEK I am going to fix this In #25 PR